docs/src/man/joins_new.md

```diff
@@ -0,0 +1,879 @@
+move forward:
+
+```jldoctest dataframe
+julia> leftjoin(df1, df2, on=:id, matchmissing=:notequal, source=:source)
+16×5 DataFrame
+ Row │ id     Saving accounts  Checking account  Risk     source
+     │ Int64  String?          String?           String?  String
+─────┼──────────────────────────────────────────────────────────────
+   1 │     9  little           moderate          bad      both
+   2 │    10  little           moderate          bad      both
+   3 │    11  little           little            bad      both
+   4 │    12  little           moderate          good     both
+   5 │    13  little           little            bad      both
+   6 │    14  little           little            good     both
+   7 │    15  moderate         little            bad      both
+   8 │     0  missing          little            missing  left_only
+   9 │     1  little           moderate          missing  left_only
+  10 │     2  little           missing           missing  left_only
+  11 │     3  little           little            missing  left_only
+  12 │     4  little           little            missing  left_only
+  13 │     5  missing          missing           missing  left_only
+  14 │     6  quite rich       missing           missing  left_only
+  15 │     7  little           moderate          missing  left_only
+  16 │     8  rich             missing           missing  left_only
+
+julia> rightjoin(df1, df2, on=:id, matchmissing=:notequal, source=:source)
+17×5 DataFrame
+ Row │ id     Saving accounts  Checking account  Risk    source
+     │ Int64  String?          String?           String  String
+─────┼──────────────────────────────────────────────────────────────
+   1 │     9  little           moderate          bad     both
+   2 │    10  little           moderate          bad     both
+   3 │    11  little           little            bad     both
+   4 │    12  little           moderate          good    both
+   5 │    13  little           little            bad     both
+   6 │    14  little           little            good    both
+   7 │    15  moderate         little            bad     both
+   8 │    16  missing          missing           good    right_only
+   9 │    17  missing          missing           good    right_only
+  10 │    18  missing          missing           bad     right_only
+  11 │    19  missing          missing           good    right_only
+  12 │    20  missing          missing           good    right_only
+  13 │    21  missing          missing           good    right_only
+  14 │    22  missing          missing           good    right_only
+  15 │    23  missing          missing           good    right_only
+  16 │    24  missing          missing           good    right_only
+  17 │    25  missing          missing           good    right_only
+```
+
+For `leftjoin` and `rightjoin` we retained `missing` but only in the table for which all 
```

Comment:
all this descriptions are incorrect and the behavior you see is unrelated with `matchmissing` kwarg value.